recipe/meta.yaml

```diff
@@ -28,13 +28,13 @@ requirements:
     - pybind11
     - boost-cpp
     - eigen
-    - cgal-cpp 5.0.1
+    - cgal-cpp 5.0.3.*
   run:
     - python
     - numpy
     - setuptools
     - boost-cpp
-    - cgal-cpp 5.0.1
+    - cgal-cpp 5.0.2.*
```

Comment:
Isn't it a bit strange to have a different version of cgal-cpp in host and run? I believe the one in run is only there to bring in the dependencies like mpfr+gmp, so its exact version is probably not important.